src/rfac/rfrmin.c

```
@@ -0,0 +1,458 @@
+        shift, experimental_data, experimental_leng, theory_data, theory_leng);
+#endif
+}
+
+/*!
+ * Returns the number of datasets in \p iv_cur array (must be terminated by
+ *
+ * \param iv_cur Pointer to #rfac_ivcur array, must be terminated by
+ * #END_OF_GROUP_ID in \c .group_id member
+ * \return Number of datasets.
+ */
+size_t rfac_ivcur_get_number_of_datasets(const rfac_ivcur *iv_cur)
```

Cppcheck (2.19.0)

[style] The function 'rfac_ivcur_get_number_of_datasets' should have static linkage since it is not used outside of its translation unit.

(staticFunction)